Source/missiles.cpp

```diff
@@ -3991,7 +3996,9 @@ void MI_Element(Missile &missile)
 		if (!CheckBlock(p, c))
 			CheckMissileCol(missile, dam, dam, true, c, true);
 
-		constexpr Displacement Offsets[] = { { 0, 1 }, { 0, -1 }, { 1, 0 }, { 1, -1 }, { 1, 1 }, { -1, 0 }, { -1, 1 }, { -1, -1 } };
+		constexpr Displacement Offsets[] = { Displacement(Direction::SouthWest), Displacement(Direction::NorthEast), 
+		                                     Displacement(Direction::SouthEast), Displacement(Direction::East), Displacement(Direction::South), 
+											 Displacement(Direction::NorthWest), Displacement(Direction::West), Displacement(Direction::North) };
 		for (Displacement offset : Offsets) {

```

Comment:
There is an implicit conversion from `Direction` to `Displacement` when adding to a point.
```suggestion
		constexpr Direction Offsets[] = {
			Direction::SouthWest,
			Direction::NorthEast,
			Direction::SouthEast,
			Direction::East,
			Direction::South,
			Direction::NorthWest,
			Direction::West,
			Direction::North,
		};
		for (Direction offset : Offsets) {
```